### ISSUE TITLE
discord bot

### ISSUE THREAD

**Issue by ghost** [sborquez/VQGAN_CLIP_docker#1]
thanks for making this, would it be possible to wrap this into a discord bot?

**Comment by sborquez** [comment]
Yes, it is possible. I could add a discord bot to generate images, but I'm not sure about the practicality; I mean, generating a new image takes about 5 to 10 minutes. Does it make sense to wait that long for the command response?
